syntax = "proto3";

option java_package = "emu.grasscutter.net.proto";
// Version: 5.0

message Weapon {
    uint32 level = 1;
    uint32 exp = 2;
    uint32 promote_level = 3;
    map<uint32, uint32> affix_map = 4;
    bool HHLNNPOILDL = 5;
}